<?xml version="1.0"?>
<launch>
    <include file="$(find assignment_2_2022)/launch/assignment1.launch" />
    <group ns="pub_speed">
        <rosparam file="$(find research_track_1)/params/pub_speed.yaml" command="load" />
    </group>
    <node pkg="research_track_1" type="goal_logger.py" name="goal_logger" output="screen" />
    <node pkg="research_track_1" type="robot_state_pub.py" name="robot_state_pub" output="screen" />
</launch>